.github/workflows/angular.yml

```diff
@@ -27,6 +27,7 @@ on:
     branches-ignore:
       - 'dependabot/**'
       - 'skip_ci*'
+      - 'spring-boot-3.0-m4'
```

Comment:
This branch have PR tests, which have more information.
Disabled on push test to avoid duplicated tests.